src/components/Book.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Book = () => (
+  <li>
+    <h3>Book name</h3>
+    <button type="button"> remove</button>
+  </li>
+);
+
+export default Book;
```

Comment:
- [ ] Kindly update your `Book` component, it should take the book data as a prop and render the title and the author.